addons/payment/wizards/payment_link_wizard_views.xml

```diff
@@ -22,14 +22,18 @@
                             attrs="{'invisible':[('has_multiple_providers', '=', False)]}"/>
                     </group>
                 </group>
-                <group>
-                    <field name="link" readonly="1" widget="CopyClipboardChar"/>
-                </group>
                 <group attrs="{'invisible':[('partner_email', '!=', False)]}">
                     <p class="alert alert-warning fw-bold" role="alert">This partner has no email, which may cause issues with some payment providers. Setting an email for this partner is advised.</p>
                 </group>
                 <footer>
-                    <button string="Close" class="btn-primary" special="cancel" data-hotkey="z"/>
+                    <field name="link"
+                           readonly="1"
+                           widget="CopyClipboardButton"
+                           options="{'label': 'Generate and Copy Payment Link'}"/>
+                    <button string="Close"
+                            class="btn btn-secondary rounded-3"
```

Comment:
Same as for the copy button.
```suggestion
                            class="btn btn-secondary rounded-2"
```

Reply:
OK. Shouldn't the Close Button be in a div as well? Just the same way a field with the Button widget is (see image)
![image](https://user-images.githubusercontent.com/112173839/197164601-9952f497-cad4-42e6-9728-195ced683d47.png)

Reply:
It could, but that wouldn't change anything. I believe it's fine if your button lives its life directly in the footer.

Reply:
I still find it weird that heights are not the same
![image](https://user-images.githubusercontent.com/112173839/197472283-4b14fa93-dda3-43dd-acd8-8487377deb96.png)